src/index.ts

```diff
@@ -1,28 +1,21 @@
 import {NgModule, ModuleWithProviders} from '@angular/core';
 
-import {NgbAccordionModule, NgbPanelChangeEvent} from './accordion/accordion.module';
+import {NgbAccordionModule} from './accordion/accordion.module';
 import {NgbAlertModule} from './alert/alert.module';
 import {NgbButtonsModule} from './buttons/buttons.module';
 import {NgbCarouselModule} from './carousel/carousel.module';
 import {NgbCollapseModule} from './collapse/collapse.module';
 import {NgbDatepickerModule} from './datepicker/datepicker.module';
 import {NgbDropdownModule} from './dropdown/dropdown.module';
-import {
-  NgbModalModule,
-  NgbModal,
-  NgbModalConfig,
-  NgbModalOptions,
-  NgbModalRef,
-  ModalDismissReasons
-} from './modal/modal.module';
+import {NgbModalModule} from './modal/modal.module';
 import {NgbPaginationModule} from './pagination/pagination.module';
 import {NgbPopoverModule} from './popover/popover.module';
 import {NgbProgressbarModule} from './progressbar/progressbar.module';
 import {NgbRatingModule} from './rating/rating.module';
-import {NgbTabsetModule, NgbTabChangeEvent} from './tabset/tabset.module';
```

Comment:
But people might use it for type checking, no? Applies to other places as well.

Reply:
These are just imports, we have all the exports a bit lower in the file: https://github.com/ng-bootstrap/ng-bootstrap/blob/8dc0513fa630df22faf16fdaae26515206644b05/src/index.ts#L88-L96